app/scripts/modules/core/src/application/config/applicationConfig.controller.js

```diff
@@ -64,5 +65,14 @@ module.exports = angular
           this.notifications = notifications;
         });
       };
+
+      if (this.feature.managedResources) {
+        this.hasManagedResources = false;
+        ManagedReader.getApplicationSummary(this.application.name).then(({ hasManagedResources }) => {
+          $scope.$applyAsync(() => {
```

Comment:
recording an in-person chat for posterity: we have no idea why. it's probably because of Some Subtle State Bug(tm) that popped up and we've be copy-pasting the same logic ever since.